### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by TheTakylo** [Songkeys/Translateer#10]
I have this error message, it is not present all the time but this time even by restarting the docker 5 times it does not work

```
{
  "statusCode": 500,
  "error": "Internal Server Error",
  "message": "Waiting for selector `div[aria-live=polite] span` failed: Waiting failed: 30000ms exceeded"
}
```

**Comment by therealsamsquanch** [comment]
I've seen it a few times, it happens when I make requests too fast

**Comment by TheTakylo** [comment]
did you manage to solve it? even rebuilding the docker it doesn't work, I had to reinstall my virtual machine entirely.

**Comment by therealsamsquanch** [comment]
Try to send fewer requests per minute
